src/test/groovy/ee/tuleva/onboarding/OnboardingServiceApplicationSpec.groovy

```diff
@@ -7,12 +7,21 @@ import spock.lang.Specification
 
 @SpringBootTest
 class OnboardingServiceApplicationSpec extends Specification {
-
     @Autowired
     WebApplicationContext context
 
     def "context loads"() {
         expect:
         context != null
     }
+
+    def "test"() {
```

Comment:
This doesn't really test anything

Reply:
I'm not really sure what happened here. This wasn't supposed to be committed :sweat_smile: